clang/include/clang/Basic/Attr.td

```diff
@@ -1957,6 +1957,16 @@ def : MutualExclusions<[SYCLIntelFPGAIVDep,
 def : MutualExclusions<[SYCLIntelFPGAMaxConcurrency,
                         SYCLIntelFPGADisableLoopPipelining]>;
 
+def SYCLIntelFPGALoopCountAvg : StmtAttr {
+  let Spellings = [CXX11<"intel","loop_count_avg">];
+  let Subjects = SubjectList<[ForStmt, CXXForRangeStmt, WhileStmt, DoStmt],
+                             ErrorDiag, "'for', 'while', and 'do' statements">;
+  let Args = [ExprArgument<"NTripCount">];
+  let LangOpts = [SYCLIsDevice, SilentlyIgnoreSYCLIsHost];
+  let HasCustomTypeTransform = 1;
```

Comment:
Is this needed? The comments on that flag say it applies to type attributes but this is a statement attribute.

Reply:
I still don't think this is needed -- the only CI failure I saw when this was removed was a test timeout that seems entirely unrelated.

Reply:
Agreed with @Aaron that is actually needed.